code/__DEFINES/flags.dm

```diff
@@ -19,11 +19,7 @@
 #define OVERLAY_QUEUED  256		//atom queued to SSoverlay
 #define ON_BORDER		512		// item has priority to check when entering or leaving
 
-#define EARBANGPROTECT		1024
-
-#define NOSLIP		1024 		//prevents from slipping on wet floors, in space etc (NOTE: flag shared with THICKMATERIAL for external suits and helmet)
-
-#define HEADBANGPROTECT		4096
+#define NOSLIP		1024 		//prevents from slipping on wet floors, in space etc

```

Comment:
But it doesn't share with THICKMATERIAL. THICKMATERIAL is 8192, which it shares with BLOCK_GAS_SMOKE_EFFECT.